ocp_resources/serving_runtime.py

```diff
@@ -0,0 +1,94 @@
+from typing import List, Optional, Any
+
+from ocp_resources.resource import NamespacedResource
+
+
+class ServingRuntime(NamespacedResource):
+    """
+    https://kserve.github.io/website/master/modelserving/servingruntimes/
```

Comment:
some args in the code (e.g `protocolVersions`) do not appear under https://kserve.github.io/website/master/modelserving/servingruntimes/#spec-attributes, can you please update the link to the API  in github?